pages/reset-password.test.ts

```diff
@@ -0,0 +1,23 @@
+import { mount, enableAutoUnmount } from '@vue/test-utils'
+import { getByClass } from 'root/testUtils'
+import resetPassword from 'pages/reset-password.vue'
+
+function getResetPasswordWrapper() {
+  return mount(resetPassword)
+}
+
+vi.stubGlobal('definePageMeta', () => {})
+
+enableAutoUnmount(afterEach)
+
+describe.skip('/reset-password?code=<RESET_PASSWORD_CODE>', () => {
```

Comment:
what is this skip function?

Reply:
Just skipping the test suite. I think I want to implement cypress or playwright, cause these page tests kinda suck just using the `@vue/test-utils`